src/test/java/org/jabref/logic/cleanup/RemoveLinksToNotExistentFilesTest.java

```diff
@@ -64,14 +83,53 @@ void setUp(@TempDir Path bibFolder) throws IOException {
 
     @Test
     void deleteLinkedFile() {
+
+        assertEquals(2, entry.getFiles().size()); //Entry has 2 linked files
+
         fileBefore.toFile().delete();
+        
         List<FieldChange> changes = removeLinks.cleanup(entry);
+        
         assertFalse(changes.isEmpty());
+        
+        assertEquals(1, entry.getFiles().size());
+        assertTrue(entry.getFiles().get(0).isOnlineLink()); //The non-deleted file should be linked online
+
+
+        BibEntry expectedEntry = new BibEntry(StandardEntryType.Article)
+                .withField(StandardField.AUTHOR, "Shatakshi Sharma and Bhim Singh and Sukumar Mishra")
+                .withField(StandardField.DATE, "April 2020")
+                .withField(StandardField.YEAR, "2020")
+                .withField(StandardField.DOI, "10.1109/TII.2019.2935531")
+                .withField(StandardField.FILE, FileFieldWriter.getStringRepresentation(
+                    new LinkedFile("", "https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=8801912:PDF", "")))
+                .withField(StandardField.ISSUE, "4")
+                .withField(StandardField.ISSN, "1941-0050")
+                .withField(StandardField.JOURNALTITLE, "IEEE Transactions on Industrial Informatics")
+                .withField(StandardField.PAGES, "2346--2356")
+                .withField(StandardField.PUBLISHER, "IEEE")
+                .withField(StandardField.TITLE, "Economic Operation and Quality Control in PV-BES-DG-Based Autonomous System")
+                .withField(StandardField.VOLUME, "16")
+                .withField(StandardField.KEYWORDS, "Batteries, Generators, Economics, Power quality, State of charge, Harmonic analysis, Control systems, Battery, diesel generator (DG), distributed generation, power quality, photovoltaic (PV), voltage source converter (VSC)");
```

Comment:
This should probably be defined in the beginning of the function if we want to follow the Arrange, Act, Assert structure.